src/main/java/io/jenkins/plugins/orka/AgentTemplate.java

```diff
@@ -68,7 +69,7 @@
     public AgentTemplate(String vmCredentialsId, String vm, boolean createNewVMConfig, String configName,
             String baseImage, int numCPUs, int numExecutors, String remoteFS, Mode mode, String labelString,
             String namePrefix, RetentionStrategy<?> retentionStrategy, OrkaVerificationStrategy verificationStrategy,
-            List<? extends NodeProperty<?>> nodeProperties) {
+            List<? extends NodeProperty<?>> nodeProperties, String jvmOptions) {
```

Comment:
To keep backwards compatibility we should introduce a new ctor and keep the old one.